srcpkgs/ffcast/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ffcast'
+pkgname=ffcast
+version=2.5.0
+revision=1
+wrksrc=FFcast-${version}
+build_style=gnu-configure
+hostmakedepends="autoconf automake"
+depends="bash ffmpeg ImageMagick xdpyinfo xprop xwininfo"
+short_desc="Run command on rectangular screen regions"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/lolilolicon/FFcast"
+distfiles="https://github.com/lolilolicon/FFcast/archive/${version}.tar.gz"
+checksum=09ebf44f76a1813eb8db78df0256f7a1cd259562cf87908b6d683f7fd1ec6b30
+
+pre_configure() {
+	./bootstrap
+}
+
+do_configure() {
+	./configure --prefix /usr --libexecdir /usr/lib
+}
+
+do_build() {
+	make
+}
+
+do_install() {
+	make install
+}
```

Comment:
You don't need (or want) these. Just let the build style do its thing.

Reply:
Sorry about that. I have removed those lines. Thank you very much.